src/test/java/com/epam/training/center/hw1/CalculatorTests/AbstractCalculatorTest.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.training.center.hw1.CalculatorTests;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public abstract class AbstractCalculatorTest {
+
+    protected Calculator calculator;
+
+    @BeforeMethod
```

Comment:
Are you sure?

This method will be called each time test method will run. So you will have init and tearDown too much times. Rewrite.

Please, read about test annotation again very carefully:
https://habr.com/ru/company/otus/blog/450872/